option/User/setup.sh

```diff
@@ -14,6 +14,19 @@ pw_create_account ( ) {
     $CHOWN $UGID ${BOARD_FREEBSD_MOUNTPOINT}${HOME_DIR}/$1
 }
 
+pw_add_user_to_group ( ) {
+    echo "Adding user $1 to group $2"
+    $PW -V ${BOARD_FREEBSD_MOUNTPOINT}/etc/ groupmod $2 -m $1
+}
+
 # Add the specified account.
-strategy_add $PHASE_FREEBSD_BOARD_INSTALL pw_create_account $1
+# Ensure it happens before adding user to groups
+PRIORITY=110 strategy_add $PHASE_FREEBSD_BOARD_INSTALL pw_create_account $1
```

Comment:
In that case, setting PRIORITY here makes sense.  But you should add a comment to document that it must happen after package installation.